cmd/ocm/create/cluster/cmd.go

```diff
@@ -56,14 +58,15 @@ var args struct {
 	private               bool
 	multiAZ               bool
 	ccs                   c.CCS
+	byovpc                c.BYOVPC
```

Comment:
please avoid using `byo-vpc` terminology and use existing vpc instead

Reply:
done